ecomap/src/python/ecomap/config.py

```diff
@@ -0,0 +1,52 @@
+from ConfigParser import SafeConfigParser
+from time import time
+import logging
+
+password = ['password', 'pass', 'pswd']          # keys which must be string
+logger = logging.getLogger('example')
+logging.basicConfig(level=logging.DEBUG)
+timeToUpdate = 15 * 60                           # 15 minutes
```

Comment:
Please use for method, variable next format:
Constants:
REFRESH_TIME
class name:
Config or ConfigParser
method:
_parse_confs
variable:
self.create_time